src/components/profile/EditProfile.js

```diff
@@ -145,7 +146,7 @@ const EditProfile = ({ screenProps, styles, navigation }) => {
     push(`Dashboard`)
   }, [push])
 
-  const handleAvatarPress = useCallback(
+  const handleAvatarPress = useOnPress(
```

Comment:
1. why do you call preventDefault if you're using useOnPress ?
2. you skipped useOnPress in the next one handleCameraPress